### PR TITLE
two windings and three windings transformers have only one site in ex…

### PR DESCRIPTION
**Pull request by basseche**
P.S : we don't migrate old filters with mistakes on sites

### CODE REVIEW

File: src/components/filter/expert/expertFilterConstants.ts [hunk 1]
```diff
@@ -851,13 +851,6 @@ export const FIELDS_OPTIONS = {
         valueEditorType: 'select',
         defaultValue: '',
     },
-    SUBSTATION_PROPERTY_3: {
-        name: FieldType.SUBSTATION_PROPERTY_3,
-        label: 'substationProperty3Twt',
```
Comment (Mathieu-Deharbe) [inline src/components/filter/expert/expertFilterConstants.ts:856]: I am quite sure that you can remove the associated loc : 
* substationProperty3Twt: 'Substation property 3',
* substationProperty3Twt: 'Propriété site 3',

**Review by Mathieu-Deharbe — CHANGES_REQUESTED** [review]
I tried to apply a filter using SUBSTATION_PROPERTIES and I got this error : 

image

I am afraid that SUBSTATION_PROPERTIES and SUBSTATION_PROPERTIES_1 are different in the back and that you might have to modify it there.

You'll probably also have to "migrate" the old SUBSTATION_PROPERTIES_1 into SUBSTATION_PROPERTIES.

File: src/components/filter/expert/expertFilterConstants.ts (hunk 1 above)
Comment (EtienneLt) [reply src/components/filter/expert/expertFilterConstants.ts:856]: in both translations

Comment (basseche) [reply src/components/filter/expert/expertFilterConstants.ts:856]: done

**Review by Mathieu-Deharbe — APPROVED** [review]

**Review by EtienneLt — APPROVED** [review]

**Pull request merged**